docs/guide/debugging-c-cpp-controllers.md

```diff
@@ -118,5 +118,8 @@ $1 = 0x0
 
 The `frame` command instructs the debugger to select the specified stack frame, and the `print` command prints the current value of an expression.
 In this simple example we clearly see that the problem is caused by a NULL (0x0) *time\_string* argument passed to the `sprintf` function.
-The next steps are to: fix the problem, recompile the controller and reload the world to give it another try.
-Once it works correctly you can remove the *-g* flag from the Makefile.
+The next steps are to: 
+1. Fix the problem
+2. Recompile the controller 
+3. Reload the world to give it another try.
+Once it works and gives the correct output you can remove the *-g* flag from the Makefile.
```

Comment:
```suggestion

Once it works and gives the correct output you can remove the *-g* flag from the Makefile.
```
Otherwise, it will appear on the same line than point 3.

Reply:
Thanks for guiding. I'll correct this error.